app/src/controllers/loading/reducer.js

```diff
@@ -0,0 +1,24 @@
+import { FETCH_START, FETCH_SUCCESS } from 'controllers/fetch/constants';
+import { NAMESPACE as LAUNCH_NAMESPACE } from 'controllers/launch/constants';
+import { NAMESPACE as FILTER_NAMESPACE } from 'controllers/filter/constants';
+import { NAMESPACE as MEMBERS_NAMESPACE } from 'controllers/members/constants';
+
+export const loadingReducer = (state = false, { type, meta }) => {
+  if (
+    meta &&
+    meta.namespace &&
+    meta.namespace !== LAUNCH_NAMESPACE &&
```

Comment:
This reducer will affect many different places at once. To prevent this, you need to add target namespace to it:

```
export const loadingReducer = (namespace) => (state = false, { type, meta }) => {
  if (meta && meta.namespace && meta.namespace !== namespace) {
    return state;
  }
  // ...
```

and the add it to the place where you want to use it (controllers/launch/reducer, controlers/filter/reducer)

A global one will be needed for adding progressbar to the top of the screen, but it's out of scope for this task.